tests/integrationtest/r/globalindex/mem_index_lookup.result

```diff
@@ -7,20 +7,56 @@ CREATE TABLE `t` (
 UNIQUE KEY `idx1` (`b`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin
 PARTITION BY HASH (`a`) PARTITIONS 5;
+insert into t values (1, 2), (2, 3), (3, 4), (4, 5);
 begin;
-insert into t values (1, 2), (2, 3), (3, 4), (4, 5), (5, 1);
+insert into t values (5, 1);
 explain select * from t use index(idx1) where b > 2;
 id	estRows	task	access object	operator info
-Projection_5	3333.33	root		globalindex__mem_index_lookup.t.a, globalindex__mem_index_lookup.t.b
-└─UnionScan_6	3333.33	root		gt(globalindex__mem_index_lookup.t.b, 2)
-  └─IndexLookUp_9	3333.33	root	partition:all	
-    ├─IndexRangeScan_7(Build)	3333.33	cop[tikv]	table:t, index:idx1(b)	range:(2,+inf], keep order:false, stats:pseudo
+Projection_5	3333.33	root	NULL	globalindex__mem_index_lookup.t.a, globalindex__mem_index_lookup.t.b
+└─UnionScan_6	3333.33	root	NULL	gt(globalindex__mem_index_lookup.t.b, 2)
+  └─IndexLookUp_11	3333.33	root	NULL	NULL
+    ├─Selection_10(Build)	3333.33	cop[tikv]	NULL	in(_tidb_pid, pid0, pid1, pid2, pid3, pid4)
+    │ └─IndexRangeScan_7	3333.33	cop[tikv]	table:t, index:idx1(b)	range:(2,+inf], keep order:false, stats:pseudo
     └─TableRowIDScan_8(Probe)	3333.33	cop[tikv]	table:t	keep order:false, stats:pseudo
 select * from t use index(idx1) where b > 2;
 a	b
 2	3
 3	4
 4	5
+explain select * from t partition(p0) use index(idx1) where b <= 2;
+id	estRows	task	access object	operator info
+Projection_5	3323.33	root	NULL	globalindex__mem_index_lookup.t.a, globalindex__mem_index_lookup.t.b
+└─UnionScan_6	3323.33	root	NULL	le(globalindex__mem_index_lookup.t.b, 2)
+  └─IndexLookUp_11	3323.33	root	NULL	NULL
+    ├─Selection_10(Build)	3323.33	cop[tikv]	NULL	in(_tidb_pid, pid0)
+    │ └─IndexRangeScan_7	3323.33	cop[tikv]	table:t, index:idx1(b)	range:[-inf,2], keep order:false, stats:pseudo
+    └─TableRowIDScan_8(Probe)	3323.33	cop[tikv]	table:t	keep order:false, stats:pseudo
+select * from t partition(p0) use index(idx1) where b <= 2;
+a	b
+5	1
+explain select * from t partition(p1) use index(idx1) where b <= 2 and a = 10;
+id	estRows	task	access object	operator info
+Projection_5	3.32	root		globalindex__mem_index_lookup.t.a, globalindex__mem_index_lookup.t.b
+└─UnionScan_6	3.32	root		eq(globalindex__mem_index_lookup.t.a, 10), le(globalindex__mem_index_lookup.t.b, 2)
+  └─IndexLookUp_12	3.32	root		
+    ├─Selection_10(Build)	3323.33	cop[tikv]		in(_tidb_pid, -1)
```

Comment:
in(_tidb_pid, -1) what does it mean?

Reply:
An invalid partition id.

Reply:
Updated, replace it to dual by https://github.com/pingcap/tidb/pull/52660/files#diff-ee13bf7b2ccb6e17bfb8c41536d4983809ee41eea0022a6d9eb27a61f6a96c8eR41